query-planner-js/src/index.ts

```diff
@@ -28,11 +29,17 @@ export class QueryPlanner {
       return { kind: 'QueryPlan' };
     }
 
-    return computeQueryPlan({
+    const {plan, statistics} = computeQueryPlan({
```

Comment:
Note for reviewers: I initially considered to keep returning the result of `computeQueryPlan` from `buildQueryPlan`, which would be admittedly slightly cleaner than having a separate `lastGeneratedPlanStatistics` method. But that `buildQueryPlan` is used in a bunch of places, including the router bridge I believe, and I was worried about create some headache (especially since this fix is meant for a patch release).